### ISSUE TITLE
Walmart specific NPM urls in package-lock file

### ISSUE THREAD

**Issue by maxnachlinger** [walmartlabs/nocktor#9]
For example:
```
    "@babel/code-frame": {
      ...snip
      "resolved": "http://npme.walmart.com/@babel/code-frame/-/code-frame-7.0.0-beta.44.tgz",
    },
```
cc @dylancwood @ChrisCrewdson

**Comment by ChrisCrewdson** [comment]
This should be corrected with https://github.com/walmartlabs/nocktor/pull/10

**Comment by ChrisCrewdson** [comment]
Merged #10.
